### PR TITLE
[WIP] Devx-292-labeling markets

### PR DESCRIPTION
**Pull request by Astro-Choco**
The PR is created to review changes for listing available markets for the SDK. See https://izettle.atlassian.net/browse/DEVX-302.
Same changes made in SDK iOS README. See https://github.com/iZettle/sdk-ios/pull/339.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,26 @@ The `SDK` is compatible with apps supporting Android API level 19 and above. The
...
+-   Sweden
+-   Brazil
+-   Finland
+-   Germany
+-   Italy
+-   Mexico
+-   Spain
+-   Netherlands
+
+You can integrate your Point of Sales (POS) with the `SDK` only for the available markets, no matter where you are located.
```
Comment (iznatalia) [inline README.md:27]: We shouldn't use word integrate here. We suggested to write `build with Zettle SDK`.

I also feel this sentence confuses more than clarifies. Maybe drop it? What do you think @Sitni ?

File: README.md [hunk 2]
```diff
@@ -8,6 +8,26 @@ The `SDK` is compatible with apps supporting Android API level 19 and above. The
 
 The `SDK` requires location permission to function properly. It is needed to scan and connect Bluetooth devices, in this case, the Zettle Readers. During the integration process, you will be required to present your Github access token and your oAuth credentials. It is best to have these in advance. 
 
+Card payments with Zettle is currently accepted in the following available markets:
```
Comment (iznatalia) [inline README.md:11]: A card payment might get rejected in a legit market for other reasons.

`following available` sounds like tautology to me.

Maybe `Card payments with Zettle SDK are currently supported in the following markets`

File: README.md (hunk 1 above)
Comment (Astro-Choco) [reply README.md]: Based on comments, I changed `available` to `supported`. We will keep `integrate` as discussed.

File: README.md (hunk 2 above)
Comment (Astro-Choco) [reply README.md]: Based on comments, changed "available" to "supported".

**Review by iznatalia — APPROVED** [review]
LGTM! 👍

**Review by PP-LaszloPinter — APPROVED** [review]

**Pull request merged**